ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/GODictionaryRenderer/GODictionaryRenderer.cpp

```diff
@@ -190,7 +190,7 @@ void GODictionaryRenderer::parseXmlFile()
         if (!currentElementName.isEmpty())
         {
           // Get the text and store it as the current element's value
-          QStringRef trimmedText = m_xmlParser.text().trimmed();
+          QStringView trimmedText = m_xmlParser.text().trimmed();
```

Comment:
Is there a guideline in samples about using `auto`? Here, `QXmlReader::text` returns `QStringView`, then it's possible to use `auto`.

And maybe `trimmedText` can be `const`?

Reply:
Yeah, unfortunately sample viewer code advises against using 'auto'. Personally I am a big fan of it myself!

Reply:
The reason for that is generally more geared towards user facing code - so that a snippet that showcases our API usage is more self explanatory. WIth this being sample viewer internals, I'm fine with using auto

Reply:
@ldanzinger This is from a sample, not the sample viewer so I'm inclined to keep it as is.